lib/pbench/server/cache_manager.py

```diff
@@ -909,16 +909,16 @@ def get_unpacked_size(self) -> int:
         if not self.dataset:
             return 0
 
-        size = Metadata.getvalue(self.dataset, Metadata.SERVER_UNPACKED)
-        self.logger.info("SIZE unpacked = {}", size)
+        source = Metadata.SERVER_UNPACKED
+        size = Metadata.getvalue(self.dataset, source)
         if not size:
             try:
-                size = int(
-                    Metadata.getvalue(self.dataset, "dataset.metalog.run.raw_size")
-                )
-                self.logger.info("SIZE metalog = {}", size)
+                source = "dataset.metalog.run.raw_size"
+                size = int(Metadata.getvalue(self.dataset, source))
```

Comment:
Why do we need a typecast at line 917 and not at line 913?

Reply:
Because `server.unpacked-size` is completely under our control and we create it as an `int` whereas the `metadata.log` value is a string.